server/views/components/sample/sample.njk

```diff
@@ -2,6 +2,6 @@
   <div class="sample-component">
     <h2 class="sample-component__heading">Sample component</h2>
     <p>{{ text }}</p>
-    {% include "views/components/grid-placeholder.njk" %}
+    {% include "views/grid/placeholder/placeholder.njk" %}
```

Comment:
It won't might make text searching for files fiddly unless you know that pattern though, which is fairly simple to figure out.